### PR TITLE
Code reviews can also be a medium to show appreciation

### PR DESCRIPTION
**Pull request by stefannibrasil**
Whenever I learn something new or someone shares something new on a PR, I feel that showing appreciation for their work is a good practice to build when doing code reviews.

I thought of adding this one to our guides. Let me know if it belongs here. Thanks!

### CODE REVIEW

File: code-review/README.md
Comment (sarahraqueld) [inline code-review/README.md:29]: (feel free to disregard this as it is very small)
How about keeping the pattern of examples in parenthesis like the items above?

```suggestion
- If you learned something new, share your appreciation. 
   ("I did not know about this. Thank you for sharing it".)
```

**Review by sarahraqueld — APPROVED** [review]
💯

**Review by stevepolitodesign — APPROVED** [review]
This is great! Thank you 👏

**Review by neilvilela — APPROVED** [review]
Great guidance! Thank you for making it a guideline.

**Review by DoodlingDev — APPROVED** [review]

**Review by thoughtbot-summer — APPROVED** [review]

**Review by JoelQ — APPROVED** [review]
💯  to this! I'm surprised we didn't have something like this already in our guide but we definitely should!

**Comment by stefannibrasil** [comment]
Thank you, all! You're awesome 🌟

**Comment by sarahraqueld** [comment]
@stefannibrasil In this case is probably fine that you merged it, but I wanted to remind you of a contributing guideline stated on the readme:

if you have commit access, please don't merge changes without waiting a week for everybody to leave feedback.

**Comment by stefannibrasil** [comment]
Oh, I completely missed that 🤦‍♀️ 

if anyone has any other feedback, please let me know, and I will update it. Apologies in advance!

**Pull request merged**
